### ISSUE TITLE
This repo is quite dated

### ISSUE THREAD

**Issue by alkuzad** [yellowdoge/mediacapture-record-implementation-status#1]
Just question to @yellowdoge, is there an resource/option to find it on official Google materials ? I'm mostly interested in VP8/VP9 support for WebRTC purposes.

**Comment by yell0wd0g** [comment]
It's a bit spread all over the crbug.com. Are you interested in a particular platform?

**Comment by alkuzad** [comment]
@yellowdoge windows / macos, I see some open tickets around them but it's hard to follow what really it touches - general platform support or media recorder support
